src/channel-list-item.tsx

```diff
@@ -14,6 +14,9 @@ import { SimpleView } from './lib/view';
 import { Store } from './lib/store';
 import { Updatable } from './lib/updatable';
 
+import {when} from './lib/when';
```

Comment:
I've been converted to the church of "Spaces Around Braces," which is the One True Church.